json-lint/test/resources/schema/inventory/valid/contentad.test.json

```diff
@@ -0,0 +1,17 @@
+{
+    "placements": {
+        "nativeAd": {
```

Comment:
This should now be `bottom_details_2` or a valid placement.

Reply:
@gianmarcobrunialtimrf you should do the rollout as well.

Reply:
@ricardoribasmrf It's been rolled out 1 hour ago. ;) Most probably Cristian had already worked on it, but since now everybody knows what to do...